packages/website/gatsby-config.js

```diff
@@ -0,0 +1,48 @@
+const path = require("path")
+
+module.exports = {
+  siteMetadata: {
+    title: `Gatsby Default Starter`,
```

Comment:
Maybe initialize the file with atti-components meta data ?

Reply:
Done